specification/digitaltwins/resource-manager/readme.go.md

```diff
@@ -0,0 +1,19 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+```
+
+### Tag: package-2020-03-01-preview and go
+
+These settings apply only when `--tag=package-2020-03-01-preview --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2020-03-01-preview' && $(go)
+namespace: Microsoft.DigitalTwins
+output-folder: $(go-sdks-folder)/digitaltwins/Generated
```

Comment:
Please change the output folder to 
`$(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-03-01-preview/$(namespace)`